app/helpers/admin/volunteers_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+	module VolunteersHelper
+		def can_manage_volunteers(conference)
+			if (current_user.has_role? :organizer, conference) || (current_user.has_role? :volunteers_coordinator, conference)
+				true
+			else
+				false
+			end
+		end
+	end
+end
```

Comment:
Also, let's put a question mark at the end of that method name: `can_manage_volunteers?`